### PR TITLE
implemented sorting of data using pipe

### PR DESCRIPTION
**Pull request by vikas-bisht**

### CODE REVIEW

File: src/app/filter.pipe.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { Todo } from './todo';
+import { ListComponent } from './list.component';
```
Comment (Rahul-Khera-Codes) [inline src/app/filter.pipe.ts]: why import list component and todo here ?

File: src/app/filter.pipe.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+@Pipe({
+  name: "sort"
+})
+export class FilterPipe implements PipeTransform {
+  transform(array: Array<any>, args: string): Array<string> {
+    {
+      array.sort((a: any, b: any) => {
```
Comment (Rahul-Khera-Codes) [inline src/app/filter.pipe.ts]: use a fine variable name, not just a/b/s/. change it

Comment (vikas-bisht) [reply src/app/filter.pipe.ts]: ok sir!

File: src/app/filter.pipe.ts (hunk 1 above)
Comment (vikas-bisht) [reply src/app/filter.pipe.ts]: forget to removed them sir!will removed!

**Pull request merged**